### DISCUSSION TITLE
Configure globally PropertyNameMappingStrategy.CaseInsensitive in .NET CORE 8

### DISCUSSION THREAD

**Discussion by holpit** [riok/mapperly#1401] [Q&A]
Hi,
I would like to set globally the CaseStrategy.
I know that there is the assembly way
[assembly: MapperDefaults(PropertyNameMappingStrategy = PropertyNameMappingStrategy.CaseInsensitive)]

but for .net core 8 is better to use the csproj file.
Do you know how to set the right configuration?

<PropertyGroup>
   <MapperDefaults>...</MapperDefaults>
  </PropertyGroup>

Thank you

**Comment by latonz** [comment]
This is not possible. Mapperly only supports the configuration with the attributes. Why do you think configuring in the csproj is `better`?

**Reply by holpit** [reply]
I mean that most of the AssemblyInfo.cs properties has the csproj equivalent and is common to use a common.props file to configure projects in .NET.
Also Mapperly has one csproj onfiguration:
`
<PropertyGroup>
  <MapperlyAbstractionsScope>runtime</MapperlyAbstractionsScope>
</PropertyGroup>
`
By the way, is not a problem. It's just a my convention to configure projects.

AssemblyInfo attribute works as expected.

Thank you.
